core/src/main/java/io/grpc/internal/ServerImpl.java

```diff
@@ -512,18 +524,23 @@ public void onReady() {}
   @VisibleForTesting
   static class JumpToApplicationThreadServerStreamListener implements ServerStreamListener {
     private final Executor callExecutor;
+    private final Executor cancelExecutor;
```

Comment:
This is no longer needed since in the test it's always the same as `callExecutor`.